dev/CommonStyles/GridViewItem_themeresources_vu.xaml

```diff
@@ -0,0 +1,190 @@
+            <StaticResource x:Key="GridViewItemCheckDisabledBrush" ResourceKey="TextOnAccentAAFillColorDisabledBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxBrush" ResourceKey="ControlAltFillColorSecondaryBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxPointerOverBrush" ResourceKey="ControlAltFillColorSecondaryBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxPressedBrush" ResourceKey="ControlAltFillColorTertiaryBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxDisabledBrush" ResourceKey="ControlAltFillColorSecondaryBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxSelectedBrush" ResourceKey="AccentAAFillColorDefaultBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxSelectedPointerOverBrush" ResourceKey="AccentAAFillColorSecondaryBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxSelectedPressedBrush" ResourceKey="AccentAAFillColorTertiaryBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxSelectedDisabledBrush" ResourceKey="AccentAAFillColorDisabledBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxBorderBrush" ResourceKey="ControlAAStrokeColorDefaultBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxPointerOverBorderBrush" ResourceKey="ControlAAStrokeColorDefaultBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxPressedBorderBrush" ResourceKey="ControlAAStrokeColorDisabledBrush" />
+            <StaticResource x:Key="GridViewItemCheckBoxDisabledBorderBrush" ResourceKey="ControlAAStrokeColorDisabledBrush" />
+            <StaticResource x:Key="GridViewItemBackgroundPointerOver" ResourceKey="GhostFillColorSecondaryBrush" />
+            <StaticResource x:Key="GridViewItemForegroundPointerOver" ResourceKey="TextFillColorSecondaryBrush" />
+            <StaticResource x:Key="GridViewItemBackgroundSelected" ResourceKey="GhostFillColorTertiaryBrush" />
+            <StaticResource x:Key="GridViewItemForegroundSelected" ResourceKey="TextFillColorPrimaryBrush" />
+            <StaticResource x:Key="GridViewItemBackgroundSelectedPointerOver" ResourceKey="GhostFillColorTertiaryBrush" />
+            <StaticResource x:Key="GridViewItemBackgroundPressed" ResourceKey="GhostFillColorTertiaryBrush" />
+            <StaticResource x:Key="GridViewItemBackgroundSelectedPressed" ResourceKey="GhostFillColorSecondaryBrush" />
+            <StaticResource x:Key="GridViewItemBackgroundSelectedDisabled" ResourceKey="GhostFillColorSecondaryBrush" />
+        </ResourceDictionary>
+        <ResourceDictionary x:Key="HighContrast">
+            <CornerRadius x:Key='GridViewItemCornerRadius'>4</CornerRadius>
+            <CornerRadius x:Key='GridViewItemCheckBoxCornerRadius'>3</CornerRadius>
+            <Thickness x:Key='GridViewItemSelectedBorderThickness'>2</Thickness>
+            <StaticResource x:Key="GridViewItemBackground" ResourceKey="GhostFillColorTransparentBrush" />
```

Comment:
I believe the plan is to keep using the old brushes for high contrast. @Yulia as FYI

Reply:
Went back to using 
`<SolidColorBrush x:Key="ListViewItemBorderBackground" Color="{ThemeResource SystemColorButtonFaceColor}" />` 
type of HighContrast resources for ListViewItem/GridViewItem.